processor/src/parallel/mod.rs

```diff
@@ -1704,3 +1970,89 @@ impl OperationHelperRegisters for TraceGenerationHelpers {
         ]
     }
 }
+
+// HELPERS
+// ================================================================================================
+
+// TODO(plafer): If we want to keep this strategy, then move the `op_eval_circuit()` method
```

Comment:
Q: Could you elaborate on this?

Reply:
We discussed offline that the fast processor could just record all the memory reads, and let the trace generator handle the evaluation

Reply:
Makes sense, thanks for the elaboration